examples/scripts/bco.py

```diff
@@ -18,6 +18,8 @@
 # Full training:
 python examples/scripts/bco.py \
     --model_name_or_path=nnheui/stablelm-2-1_6b-sft-full \
```

Comment:
Maybe we can use `Qwen2.5-0.5B-Instruct` as a demo model?

Reply:
```suggestion
    --model_name_or_path Qwen/Qwen2.5-0.5B-Instruct \
```